Example/Gemfile

```diff
@@ -1,4 +1,8 @@
 source 'https://rubygems.org'
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby '>= 2.6.10'
-gem 'cocoapods', '~> 1.12'
+
+# Cocoapods 1.15 introduced a bug which break the build. We will remove the upper
+# bound in the template on Cocoapods with next React Native release.
+gem 'cocoapods', '>= 1.13', '< 1.15'
+gem 'activesupport', '>= 6.1.7.5', '< 7.1.0'
```

Comment:
I know that Example app is quite outdated, since it's the only app that hasn't been updated to 0.73, but I'm worried that the bug might also occur there (also, example app passes `pod install` properly, so I believe we can also use those versions).